src/fabric.c

```diff
@@ -435,9 +437,12 @@ void fi_ini(void)
 	ofi_register_provider(VERBS_INIT, NULL);
 	ofi_register_provider(GNI_INIT, NULL);
 	ofi_register_provider(BGQ_INIT, NULL);
+#ifdef _WIN32
+	ofi_register_provider(NETDIR_INIT, NULL);
+#endif
```

Comment:
We shouldn't need an ifdef in this file.  This should already be handled in include/prov.h, with NETDIR_INIT set to NULL in the non-windows case.